xbmc/windowing/gbm/WinSystemGbm.cpp

```diff
@@ -246,3 +265,17 @@ void CWinSystemGbm::Unregister(IDispResource *resource)
     m_resources.erase(i);
   }
 }
+
+void CWinSystemGbm::OnLostDevice()
```

Comment:
can this belong to CWinSystem and be used by other windowing systems? it doesn't seem specific to just this windowing system.

Reply:
it's not not called from the outside and implemention is specific to implementatios. hence makes not much sense to promote it up the hierarchy